mite/cli/test.py

```diff
@@ -83,17 +86,22 @@ def test_scenarios(test_name, opts, scenarios, config_manager):
     controller = Controller(test_name, scenario_manager, config_manager)
     transport = DirectRunnerTransport(controller)
     receiver = DirectReciever()
+    meta_processor = TransactionMetadataCollector()
+    receiver.add_listener(meta_processor.process_message)
```

Comment:
This idea of adding a receiver is fundamentally ok, but why don't we just add a receiver that listens for a new message type (`curl_command`, or if you want to be generic something like `console_debug_message`) and prints out those messages as it sees them?  We don't need to accumulate metadata in a contextvar on the transaction etc etc.